pkg/corerp/api/v20231001preview/testdata/environmentresource.json

```diff
@@ -16,6 +16,19 @@
         "scope": "/planes/aws/aws/accounts/140313373712/regions/us-west-2"
       }
     },
+    "recipeConfig": {
+      "terraform": {
+        "authentication": {
+          "git": {
+            "pat": {
+              "dev.azure.com":{
+                "secretStore":"/planes/radius/local/resourcegroups/default/providers/Applications.Core/secretStores/github"
+              }
+            }
```

Comment:
What if this part doesn't exist (by mistake of the operator), would this cause a nil pointer exception?

Reply:
I dont think it will nil pointer if pat property doesnt exist

Reply:
Can we add a unit test for this?